designdocs/batched_messages.md

```diff
@@ -62,6 +62,86 @@ See the [implementation of paging in the messages pallet](https://github.com/Lib
     * `next_index`: `u32` starting index of next results in `next_block`
     * `results`: `Vec<BatchAnnouncement>`
 
+### Message Schema
+
+Distributing data on a decentralized network has a few fundamental problems that
+are worth discussing here. Primarily, those problems are:
+- What shape is this data in?
+- How do I authenticate this data?
+- How do these actors access this information?
```

Comment:
I found this confusing until reading further down.

Maybe:
`How can actors access this information?` or
`How can users access this information?`